internal/code-generator/generator/heuristics.go

```diff
@@ -164,7 +164,7 @@ func parseYaml(file string) (*Heuristics, error) {
 // - backreference
 // For referece on supported syntax see https://github.com/google/re2/wiki/Syntax
 func isUnsupportedRegexpSyntax(reg string) bool {
-	return strings.Contains(reg, `(?<`) || strings.Contains(reg, `\1`) ||
+	return strings.Contains(reg, `(?<`) || strings.Contains(reg, `(?=`) || strings.Contains(reg, `\1`) ||
```

Comment:
As not everyone may be familiar with intricacies of the regex syntax, would you mind to also updating the GoDoc comment by mentioning the meaning of this new case e.g as "the before text"/lookahead matching?

Reply:
done